signer/src/bitcoin/packaging.rs

```diff
@@ -67,13 +67,11 @@ where
     // This is a variant of the Best-Fit-Decreasing algorithm, so we sort
     // by "weight" decreasing. We use the votes against as the weight, but
     // vsize is a reasonable weight metric as well.
-    let mut item_vec: Vec<(u32, T)> = items
+    let item_vec: Vec<(u32, T)> = items
```

Comment:
The comment needs updating

Reply:
[be3aaae](https://github.com/stacks-sbtc/sbtc/pull/1664/commits/be3aaaeb3cf5d67c739e3ffcfa3402314d3e1774)